degica/komoju-woocommerce
# One-click account setup

## Context

This PR introduces a 1-click setup process that requires no explanation or manual copying and pasting. It sets up your webhook and your secret key automatically.

Here's a video of the feature working

https://user-images.githubusercontent.com/2793160/171816076-09821fdf-f03b-4f3d-9567-45bd67f1000e.mp4

## How to test

1. Visit WC settings, KOMOJU tab
2. See "sign in" or "reconnect" button (depending on whether you already have a secret key set up)
3. Click it
4. Sign into KOMOJU
5. Select which merchant you wish to connect and submit the form
6. You should now be able to select payment methods on WooCommerce
